core/nativeaot/NativeLibrary/LoadLibrary.c

```diff
@@ -60,6 +60,16 @@ int callSumFunc(char *path, char *funcName, int firstInt, int secondInt)
     #else
         void *handle = dlopen(path, RTLD_LAZY);
     #endif
+    if (!handle)
```

Comment:
Would it make sense to extract the whole library loading business into a separate function? We were already duplicating 5 lines before this change, but now it's 14 lines.